{"name": "test_mult[2-8-14]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 16 == 14\n  +16\n  -14", "trace": "self = <testcases.test_func.Test_Pytest object at 0x1068630f0>, fixture_func = <pyCode.calc.Calculator object at 0x10686bfd0>, a = 2, b = 8, result = 14\n\n    @pytest.mark.parametrize('a,b,result', test_data[2])\n    @allure.feature('乘法测试')\n    def test_mult(self, fixture_func, a, b, result):\n>       assert fixture_func.mult(a, b) == result\nE       assert 16 == 14\nE         +16\nE         -14\n\ntestcases/test_func.py:24: AssertionError"}, "parameters": [{"name": "a", "value": "2"}, {"name": "b", "value": "8"}, {"name": "result", "value": "14"}], "start": 1594609992120, "stop": 1594609992121, "uuid": "3cbcd8f2-490e-402e-b66e-12ad33b59029", "historyId": "ab20fb020a30ec47adfec0c5a0e9d658", "testCaseId": "f23b8f69bb19ec69f89b4ceb1ecccfec", "fullName": "testcases.test_func.Test_Pytest#test_mult", "labels": [{"name": "feature", "value": "乘法测试"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_func"}, {"name": "subSuite", "value": "Test_Pytest"}, {"name": "host", "value": "bogon"}, {"name": "thread", "value": "56325-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_func"}]}